src/main/java/com/monumental/models/Monument.java

```diff
@@ -19,6 +21,9 @@
 })
 public class Monument extends Model implements Serializable {
 
+    @Column(name = "slug", unique = true)
```

Comment:
Since we're utilizing the ID as a way to filter our the existing Monument in some queries, does that mean we're persisting 2 IDs? The auto-incrementing int ID and this ID?

Reply:
Yes in this setup there is both a slug and an id. I read about it a bit and I think this is actually a normal approach because slugs can change and ids can't